src/python/CRABClient/JobType/Analysis.py

```diff
@@ -146,6 +147,7 @@ def run(self, filecacheurl = None):
         with UserTarball(name=tarFilename, logger=self.logger, config=self.config) as tb:
             inputFiles = [re.sub(r'^file:', '', file) for file in getattr(self.config.JobType, 'inputFiles', [])]
             tb.addFiles(userFiles=inputFiles, cfgOutputName=cfgOutputName)
+            configArguments['adduserfiles'] = [os.path.basename(f) for f in inputFiles]
```

Comment:
rats... I spent so much time with git commands.. and yet this happened. I'll try to patch the PR